movethebytes/data-transfer/README.md

```diff
@@ -0,0 +1,275 @@
+# So you wanna write a test plan in rust for the Move The Bytes Working Group
+
+So you wanna write a test plan in rust for the Move The Bytes Working Group that tests your protocol? Here is a guide! Note: this guide assumes you already have testground installed. The purpose of this guide is to make it quicker to onboard yourself into the wonderful world of writing a test plan. 
+
+It is subject to change as we adjust our expectations for the Working Group and learn more about testground.
+
+A few things to note:
+- testground is currently not working on mac M1 architecture, so you either need to have some cloud infra you can access or a linux box at home. Or Windows I guess but that is uncharted territory GLHF
+- the dev cycle is painful. You can short circuit some issues by running `cargo run --bin MY_PLAN` to ensure it builds before running it on testground. Note: the test will panic when you run it like this, that is expected.
+- log log log. log a lot and often when you are starting to write your test plan. Logs are the only way you can get insight into your test at this stage, and the cycle to add a log and run the test again is __long__. Just err on the side of logging too much & remove everything unnecessary later.
+
+This document is written with the `github.com/n0-computer/test-plans` repository in mind.
```

Comment:
We should note that it's also slightly 🦀  centric but gophers are welcome too in this repo. But still strongly encourage 🦀 🦀 🦀  😸